src/plugins/data_view_field_editor/public/components/field_format_editor/editors/color/color.tsx

```diff
@@ -38,8 +38,8 @@ export class ColorFormatEditor extends DefaultFormatEditor<ColorFormatEditorForm
   constructor(props: FormatEditorProps<ColorFormatEditorFormatParams>) {
     super(props);
     this.onChange({
-      fieldType: props.fieldType,
-    });
+      fieldType: props.fieldType, // FIXME: why add `fieldType` as an EditorFormatParam?
+    } as unknown as ColorFormatEditorFormatParams);
```

Comment:
This is the one spot in this PR that looks odd to me: it doesn't like this `fieldType` key, so I'm not sure if maybe this is a typo in the original code (possibly should be `type` ?).

Reply:
Might be good to walk through this together.